### ISSUE TITLE
Shows `document-features` as a crate in the Rustdoc

### ISSUE THREAD

**Issue by BartMassey** [slint-ui/document-features#3]
This crate seems to insert itself into the documentation it generates. (See the lower-left corner where it appears.)

![document-features-crate](https://user-images.githubusercontent.com/117151/153096048-822a65c0-eb93-49f9-af06-88ec83b9c54f.png)

Any way to suppress that?

**Comment by tronical** [comment]
I think if you run cargo doc with just `-p rbj_eq` it might work :)

**Comment by Kijewski** [comment]
The "crates" section won't be rendered on docs.rs, see e.g. https://docs.rs/nate/0.1.10/nate/

**Comment by ogoffart** [comment]
I believe rustdoc should be called with the --no-deps option

```
cargo doc --no-deps
```

**Comment by BartMassey** [comment]
Thanks much for the time and suggestions! I'd like to be able to have full Rustdoc on my github for the crates my app actually uses, without documenting this crate which is really a doc-only dependency, though. For now I'll just continue to document features manually, I guess.

**Comment by BartMassey** [comment]
`cargo doc --no-deps` doesn't produce docs for any dependencies, yes. I want docs for my "real" dependencies, so that the docs can be standalone on Github. (I republish some items from my dependencies.)

**Comment by ogoffart** [comment]
The thing is that this is also a real dependency.

So I don't know if there are flags in rustdoc to select what dependencies to show.
Maybe you can inject JavaScript to hide it with --html-in-header or something.
